hips/utils/healpix.py

```diff
@@ -0,0 +1,42 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+"""HEALpy wrapper functions.
+
+This module contains wrapper functions around HEALPix utilizing
+the healpy library
+"""
+
+__all__ = [
+    'boundaries',
+]
+
+import healpy
+import numpy as np
+
+
+def boundaries(nside: int, pix: int, nest: bool=False) -> tuple:
+    """Returns an array containing the angle (theta and phi) in radians.
+    This method calls :py:meth:`.healpy.boundaries` and :py:meth:`.healpy.vec2ang`
```

Comment:
I think this:
```
:py:meth:`.healpy.boundaries`
```
is unnecessarily verbose, and just this will work, no?
```
`healpy.boundaries`
```
Please try to build the HTML docs locally and check if the links work.